src/test/java/io/vertx/junit5/VertxTestContextTest.java

```diff
@@ -335,10 +337,7 @@ void check_future_completion_failure() throws InterruptedException {
     VertxTestContext context = new VertxTestContext();
     context
       .assertComplete(Future.succeededFuture("bla"))
-      .compose(s -> context.assertComplete(Future.failedFuture(new IllegalStateException(s + "bla"))))
```

Comment:
Why did you remove these lines?

Reply:
Yeah, so this might be where my changes went a bit too far, in `assertComplete` I added `   assertContextNotCompletedOrFailed()`, so the second assertComplete throws. Probably makes more sense to remove that assertion from `assertComplete` and `assertFail`.